obs-studio-server/source/nodeobs_service.cpp

```diff
@@ -924,8 +924,12 @@ bool OBS_service::updateAudioStreamingEncoder() {
 	} else {
 		uint64_t trackIndex = config_get_int(ConfigManager::getInstance().getBasic(), "AdvOut", "TrackIndex");
 
+		if (audioAdvancedStreamingEncoder)
```

Comment:
after obs_encoder_release encoder is not usable anymore, can a NULL be assigned to audioAdvancedStreamingEncoder?

Reply:
It will be assigned in one of the lines below, but it's a good idea to make it null if we add more code here or modify the `if` in some way it doesn't assign a value, good point